gcp/appengine/frontend3/src/templates/vulnerability.html

```diff
@@ -82,6 +92,29 @@ <h1 class="title">{{ vulnerability.id }}</h1>
               {% endfor -%}
             </ul>
           </dd>
+          {% if vulnerability.credits -%}
+          <dt class="credits">Credits</dt>
+          <dd class="credits">
+            <ul>
+              {% for credit in vulnerability.credits -%}
+              <li class="credit">
+                <ul>
+                  <li>{{ credit.name }}{%- if 'type' in credit -%} - {{ credit.type }}{%- endif -%}</li>
```

Comment:
Can we add a space after the name and before the first "-" ?

Reply:
updated: [3c62c56](https://github.com/google/osv.dev/commit/3c62c563e73b79bd23e8eeb1c2d65a4e5faff6b5)